vllm/transformers_utils/tokenizer.py

```diff
@@ -70,18 +71,17 @@ def encode_tokens(
 
 
 def get_cached_tokenizer(tokenizer: AnyTokenizer) -> AnyTokenizer:
-    """Get tokenizer with cached properties.
-
-    This will patch the tokenizer object in place.
-
+    """
     By default, transformers will recompute multiple tokenizer properties
-    each time they are called, leading to a significant slowdown. This
-    function caches these properties for faster access."""
+    each time they are called, leading to a significant slowdown.
+    This proxy caches these properties for faster access.
+    """
+    cached_tokenizer = copy.deepcopy(tokenizer)
```

Comment:
I don't think there's any need to deep copy
```suggestion
    cached_tokenizer = copy.copy(tokenizer)
```

Reply:
You're right. Updated